helpers/toolset.sh

```diff
@@ -30,6 +30,14 @@ finalise_installation() {
     rm -f .partial
 }
 
+# Checks how many cores are available
```

Comment:
# Figures out the number of CPU cores which are available on the current
# machine.
# Globals:
#     None
# Arguments:
#     None
# Returns:
#     Number of available cores